### PR TITLE
add release script for debian to be called by jenkins

### PR DESCRIPTION
**Pull request by mikedanese**
Run by https://github.com/kubernetes/test-infra/pull/622

cc @errordeveloper @luxas

### CODE REVIEW

**Comment by mikedanese** [comment]
Once these are published, I will scope these down on my workstation and release them on every change.

File: debian/jenkins.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o nounset
+set -o xtrace
+
+
+declare -r BUILD_TAG="$(date '+%y%m%d%H%M%S')"
+declare -r IMG_NAME="debian-builder-${DEB_CHANEL}:${BUILD_TAG}"
```
Comment (spxtr) [inline debian/jenkins.sh]: channel!

Comment (mikedanese) [inline debian/jenkins.sh]: https://github.com/kubernetes/release/pull/103/files

Comment (mikedanese) [inline debian/jenkins.sh]: fixed

**Pull request merged**